.github/workflows/web-test.yaml

```
@@ -0,0 +1,105 @@
+        run: |
+          python3 -m http.server 8080 --directory sample-test/src/main/resources/web-pages &
+          echo $! > server.pid
+          sleep 2
+
+      - name: Setup Chrome and ChromeDriver
+        run: |
+          # Install Chrome if not present
+          if ! command -v google-chrome &> /dev/null; then
+            brew install --cask google-chrome
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)